aws/logs_monitoring/lambda_function.py

```diff
@@ -188,7 +217,8 @@ def compileRegex(rule, pattern):
 DD_CUSTOM_TAGS = "ddtags"
 DD_SERVICE = "service"
 DD_HOST = "host"
-DD_FORWARDER_VERSION = "2.0.0"
+DD_FORWARDER_VERSION = "2.1.0"
```

Comment:
Not sure, but 2.1.0 could have already existed. Perhaps it's better update version in a separate PR before publishing to SAM repo?

Reply:
ah yes you're right, it's currently 2.1.0 https://github.com/DataDog/datadog-serverless-functions/blame/1a7cda80b13496f7fa0192475861f6f535bdf49f/aws/logs_monitoring/lambda_function.py#L204